test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
```

Comment:
Good to see you attempting the previous LO. However, please avoid accumulating unrelated changes into a single PR. 

Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.